src/components/authLayout.tsx

```diff
@@ -7,12 +7,21 @@ type AuthLayoutProps = {
 
 export default function AuthLayout({ children, title }: AuthLayoutProps) {
   return (
-    <div className="flex min-h-screen items-center justify-center bg-patina-50">
-      <div className="card w-200 rounded-xl bg-patina-100 p-6 shadow-lg">
-        <h2 className="mb-4 text-left text-2xl font-bold text-patina-600">
-          {title}
-        </h2>
-        {children}
+    <div className="flex min-h-screen flex-col items-center justify-center bg-gradient-to-b from-patina-200 to-white px-4 py-10 sm:px-6 lg:px-8">
+      <div className="relative flex w-full max-w-4xl overflow-hidden rounded-xl bg-patina-100 shadow-lg">
```

Comment:
ماشاءالله آقا کوشا. هر بار زیبا تر از دفعه قبل :fire: 

فقط بین عرض ۴۶۴ تا ۶۴۰ پیکسل هنوز همونجوری میشه (یه تیکه خالی)
این w-full اینجا نباشه درست میشه
```suggestion
      <div className="relative flex max-w-4xl overflow-hidden rounded-xl bg-patina-100 shadow-lg">
```

بعد این مرج کنید. :rocket: